src/main/java/pageObjects/Licenses.java

```diff
@@ -0,0 +1,264 @@
+        expirationDate.sendKeys((EXPIRATION_DATE_WS));
+        saveLicense.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSES_URL));
+    }
+
+    public void checkIfNameFieldIsMandatory() {
+        signInAsAdmin();
+        licenseButton.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("add-license")));
+        addLicenseButton.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSE_DETAILS_URL));
+        saveLicense.click();
+    }
+
+    public void createLicenseWithWhiteSpacesNameField() {
+        signInAsAdmin();
+        licenseButton.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSES_URL));
+        addLicenseButton.click();
+        nameField.sendKeys(WHITE_SPACE_NAME_WS);
+        purchaseDate.sendKeys(PURCHASE_DATE_WS);
+        expirationDate.sendKeys(EXPIRATION_DATE_WS);
+        saveLicense.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("error_toaster")));
+    }
+
+    public void addLicenseTooLongName() {
+        signInAsAdmin();
+        licenseButton.click();
+        addLicenseButton.click();
+        nameField.sendKeys(LICENSE_NAME_IS_TOO_LONG);
+        saveLicense.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("error_toaster")));
+    }
+
+    public void createLicenseWithPackType() {
+        signInAsAdmin();
+        licenseButton.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("add-license")));
+        addLicenseButton.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSE_DETAILS_URL));
+        nameField.click();
+        nameField.sendKeys(LICENSE_NAME_WS);
+        licenseTypeList.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("license_type-selectList-pack")));
+        selectPackList.click();
+        purchaseDate.click();
+        purchaseDate.sendKeys(Keys.CONTROL + "a");
+        purchaseDate.sendKeys(Keys.DELETE);
+        purchaseDate.sendKeys((PURCHASE_DATE_WS));
+        expirationDate.click();
+        expirationDate.sendKeys(Keys.CONTROL + "a");
+        expirationDate.sendKeys(Keys.DELETE);
+        expirationDate.sendKeys((EXPIRATION_DATE_WS));
+        saveLicense.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSES_URL));
+    }
+    public void createLicenseWithIndividualType() {
+        signInAsAdmin();
+        licenseButton.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("add-license")));
+        addLicenseButton.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSE_DETAILS_URL));
+        nameField.click();
+        nameField.sendKeys(LICENSE_NAME_WS);
+        licenseTypeList.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("license_type-selectList-individual")));
+        selectIndividualList.click();
```

Comment:
is this a list? if its not, we should just rename it to selectIndividualType or selectIndividualOption